src/models/interfaces/IActions.ts

```diff
@@ -3,13 +3,14 @@ export const KeyboardPlatforms = [
 	'Fire TV',
 	'Roku',
 	'Kodi',
+	'Sony BRAVIA'
 ] as const;
 export type KeyboardPlatform = (typeof KeyboardPlatforms)[number];
 export const Platforms = [
 	...KeyboardPlatforms,
```

Comment:
So it is an Android TV, but also has it's own separate API that can be used to send commands?

Reply:
Correct, this TV runs Android, has an ADB interface, but is not compatible with the Android TV command set, eg

> Unsupported command: POWER, list of available commands: Home, Return, Num1, Num2, Num3, Num4, Num5, Num6, Num7, Num8, Num9, Num0, DOT, VolumeUp, VolumeDown, Mute, TvPower, EPG, Confirm, ChannelUp, ChannelDown, Up, Down, Left, Right, Display, SubTitle, Audio, MediaAudioTrack, Jump, Exit, Tv, Input, TvInput, Red, Green, Yellow, Blue, Teletext, Stop, Rewind, Forward, Prev, Next, Play, Rec, Pause, OneTouchView, GooglePlay, Netflix, ActionMenu, ApplicationLauncher, Help, Wide, SyncMenu

Reply:
It's nice that it dumps a list of usable commands. I see Netflix is in there, can you add that to the sources and test that it works? It 's fine if it uses `remote.send_command` while the other sources use `media_player.play_media`.